packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1036,7 +1128,13 @@ class RawScrollbar extends StatefulWidget {
   ///     scroll view associated with the parent [PrimaryScrollController].
   ///   * [PrimaryScrollController], which associates a [ScrollController] with
   ///     a subtree.
+  ///
+  /// This is deprecated, [thumbVisibility] should be used instead.
   /// {@endtemplate}
+  @Deprecated(
+    'Use thumbVisibility instead. '
+    'This feature was deprecated after v2.9.0-1.0.pre.',
```

Comment:
Why do they have different deprecated versions?

Reply:
Probably a copy paste mistake on my part. I wrote the original PR in December, just missed updating it.